README.md

```diff
@@ -27,18 +27,44 @@ older versions may be possible, but is not tested or documented here.
 
     curl --insecure --location --request GET 'https://localhost:8443/fineract-provider/api/v1/runreports/Expected%20Payments%20By%20Date%20-%20Formatted?R_endDate=2013-04-30&R_loanOfficerId=-1&R_officeId=1&R_startDate=2013-04-16&output-type=PDF&R_officeId=1' --header 'Fineract-Platform-TenantId: default' --header 'Authorization: Basic bWlmb3M6cGFzc3dvcmQ='
 
-The API call (above) will fail on the server (see log) due to an intern error (somehow
-the SQL query in that particular report is currently actually broken), but this illustrates
-that the integration of Pentaho as a Fineract Plugin basically works.
-([FINERACT-1176](https://issues.apache.org/jira/browse/FINERACT-1176) tracks improving API response.)
```

Comment:
If this issue is still open, how does the API call works? If it is fixed, maybe we can mark this issue as resolved.

Reply:
https://issues.apache.org/jira/browse/FINERACT-1176 - i have added some comment, and i would wish @vorburger  to close or advise about the issue